server/Server.Tests/_snapshots/api-spec.json

```diff
@@ -72,20 +72,20 @@
       }
     },
     "/customer/delete": {
-      "delete": {
+      "post": {
```

Comment:
Not the most RESTful thing I've ever done, but I'm not playing with query strings. For some reason the api client just wasn't having any off it